client/src/modules/SignIn/components/ContentSignIn/components/FormSignIn/index.js

```diff
@@ -0,0 +1,106 @@
+import React, { Fragment, useState } from 'react';
+import SignInSchema from './validate';
+import { makeStyles } from '@material-ui/core/styles';
+import InputAdornmentPassword from './components/InputAdornmentPassword';
+import { MyTextField, MyCheckbox } from '../../../../../../shared/Fields';
+import { Button } from '@material-ui/core';
+import { Formik, Form } from 'formik';
+
+const useStyles = makeStyles(theme => ({
+  form: {
+    width: '100%',
+    marginTop: theme.spacing(1),
+  },
+  submit: {
+    margin: theme.spacing(3, 0, 2),
+  },
+}));
+
+const FormSignIn = ({ handleClose }) => {
+  const classes = useStyles();
+  const [values, setValues] = useState({
+    showPassword: false,
+  });
+
+  const handleClickShowPassword = () => {
+    setValues({ ...values, showPassword: !values.showPassword });
+  };
+
+  const handleMouseDownPassword = event => {
+    event.preventDefault();
+  };
+
+  return (
+    <Fragment>
+      <Formik
+        initialValues={{
+          email: '',
+          password: '',
+          remember: false
+        }}
+        validationSchema={SignInSchema}
+        onSubmit={(values, { setSubmitting, resetForm }) => {
+          setTimeout(() => {
+            alert(JSON.stringify(values, null, 2));
+            setSubmitting(false);
+            resetForm();
+            handleClose();
+          }, 400);
+        }}
+      >
+        {({isSubmitting}) => (
+          <Form className={classes.form} noValidate>
+            <MyTextField
```

Comment:
Move all static properties to MyTextField component

Reply:
I can't change MyTextField because it is shared.
It is sugar - mix Formik and TextField from material.ui.
So when I change it with static properties like "variant: outline or fullwidth", no one can use it once more.
I can only create Wrapper with static properties for only my Fileds in component Auth. And it will be use TextField.